doc/reference/tt_cli/start.rst

```diff
@@ -0,0 +1,36 @@
+.. _tt-start:
+
+Starting a Tarantool instance
+=============================
+
+..  code-block:: bash
+
+    tt start INSTANCE
+
+``tt start`` starts the specified Tarantool :ref:`instance <admin-instance_file>`.
+
+Details
+-------
+
+``INSTANCE`` is a name of an instance from the ``instances_available`` directory
+specified in the :ref:`tt configuration file <tt-config_file_app>`. This can be:
+
+*   a name of an :ref:`instance file <admin-instance_file>` without the ``.lua`` extension.
+*   a name of a directory containing the ``init.lua`` file.
```

Comment:
```suggestion
``INSTANCE`` is the name of an instance from the ``instances_available`` directory
specified in the :ref:`tt configuration file <tt-config_file_app>`. This can be:

*   the name of an :ref:`instance file <admin-instance_file>` without the ``.lua`` extension.
*   the name of a directory containing the ``init.lua`` file.
```
Each instance has _only one_ name, so "the" is applicable.